packages/flutter_tools/lib/src/devtools_launcher.dart

```diff
@@ -57,6 +57,7 @@ class DevtoolsServerLauncher extends DevtoolsLauncher {
         _dartExecutable,
         'devtools',
         '--no-launch-browser',
+        // TODO(bkonyi): does this need to be behind the flag?
```

Comment:
If this wasn't behind the flag would it always print? We don't want the DTD URI to print to CLI in the default case, since this will not be useful for most users.

Reply:
I honestly can't remember what this comment was here for. I'll just remove it.